src/config.ts

```diff
@@ -2,5 +2,8 @@ import dotenv from 'dotenv';
 
 const env = process.env.NODE_ENV || 'development';
 
+export const WEBAPP_URL =
+  env === 'development' ? 'http://localhost:5002' : 'https://app.daily.dev';
```

Comment:
This already defined somewhere I think

Reply:
We only have one for the gateway, for the webapp, it has not been defined yet after searching through the repo.

Reply:
See here:
https://github.com/dailydotdev/daily-api/blob/32e4367fb2405f8204171b368d5953c7b2618b85/src/common/links.ts#L2

It's the same link. Please consolidate :)